extension/src/popup/components/accountHistory/HistoryItem/index.tsx

```diff
@@ -166,7 +168,7 @@ export const HistoryItem = ({
         setBodyComponent(
           <>
             {paymentDifference}
-            {formatAmount(new BigNumber(amount).toFixed(2, 1))} {destAssetCode}
```

Comment:
So right now, if you send "1 XLM" using classic payments in Freighter, that item in history will show up as `Sent 1.00 XLM`.
This would differ to the formatting that we're doing in token cases, where the SAC payment would show up as `Sent 1 XLM`.
I'm curious why we round to the 2 place decimal in the classic payment case?
I would think that we would want to show the decimals and maybe remove trailing zeros for readability(what we do for tokens now).
@piyalbasu

Reply:
I think this was an oversight as we added `.toFixed` to cut strings like `1.91111` to `1.91`. But we should remove trailing zeroes if applicable here